src/dataset.jl

```diff
@@ -9,7 +9,7 @@ function dataset(package_name::AbstractString, dataset_name::AbstractString)
 
     rdaname = joinpath(basename, string(dataset_name, ".rda"))
     if isfile(rdaname)
-        return load(rdaname)[dataset_name]
+        return load(File{format"RData"}(rdaname))[dataset_name]
```

Comment:
Thanks, I didn't know this! Maybe it could be helpful to add a note about this minimum requirement to the docs, similar to the Julia compat notes? So the following should be fine, I assume?
```suggestion
        if isdefined(FileIO, :action)
            # FileIO >= 1.6
            return load(File{format"RData"}(rdaname))[dataset_name]
        else
            # FileIO < 1.6
            return load(File(format"RData", rdaname))[dataset_name]
        end
```

Reply:
Yep, that's it! And that was a great suggestion re a compat note, thanks.